generator/wrapper_gen.py

```diff
@@ -189,7 +189,7 @@ def _str(instance, properties):
                 result += p + ': dict, len={}'.format(len(prop))
             else:
                 result += p + ': '
-            value = str(prop).replace('\\n', '\\n  ')
+            value = '\\n  '.join(str(prop).splitlines())
```

Comment:
why do you use ```\\n``` and not simple ```\n```?

Reply:
Please replace all explicit `'\\n'` with use of `os.linesep` variable for portability.